### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by mariiaovs**

### CODE REVIEW

File: pages/api/auth/[...nextauth].js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import NextAuth from "next-auth";
+import GithubProvider from "next-auth/providers/github";
+import CredentialsProvider from "next-auth/providers/credentials";
+
+export const authOptions = {
+  // Configure one or more authentication providers
```
Comment (FelineHuhn) [inline pages/api/auth/[...nextauth].js:6]: I don't know if you need this comment and the below one, or?

**Review by FelineHuhn — APPROVED** [review]
I have only looked at the files relating to the log in (as discussed). Unfortunately, I am not the biggest expert in the NextAuth field, but everything looked conclusive to me and works as intended. 

Great job 👏  Next time you can hopefully do without such a degenerate rebasing, but it's great that you were able to resolve the conflicts on your own! That's what problem solving looks like.

File: components/MultiselectContainer.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+  function openDropdown(event) {
+    event.stopPropagation();
+    setIsDropdownOpen(true);
+  }
+
+  return (
+    <StyledContainer
+      onClick={openDropdown}
+      onBlur={closeDropdown}
```
[the remark this answers is not in the document]
Comment (PaulJohannFlade) [reply components/MultiselectContainer.js:45]: Yes, it works as expected.

File: pages/api/auth/[...nextauth].js (hunk 1 above)
Comment (PaulJohannFlade) [reply pages/api/auth/[...nextauth].js:6]: I have left this comment in for myself as I want to add more providers later...

File: components/MultiselectContainer.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mariiaovs) [reply components/MultiselectContainer.js:46]: We made the `div` focusable with tabIndex="0".

**Pull request merged**